main.go

```diff
@@ -18,6 +17,6 @@ func main() {
 	w.Resize(fyne.NewSize(700, 400))
 	w.SetMaster()
 
-	updater.Enable(a, w)
+	//updater.Enable(a, w)
```

Comment:
Yep, im newbie in go.

Reply:
No worries. I can either guide you though how to get it set up or implement the required changes myself in a week or two. What would you prefer?

Reply:
Thank you, please show me the place where the update function should be I will try to close the task completely

Reply:
What you basically need to do is to wrap the `updater.Enable(a, w)` call within a check for the new setting that you added. If update checking on startup is enabled, you call the function. Otherwise you don't.

Something like this for example (though I am going to suggest a better name than `UpdateRadio` in my upcoming PR review):
```
checkForUpdates := a.Preferences().BoolWithFallback("UpdateRadio", true)
if checkForUpdates {
   updater.Enable(a, w)
}
```

Reply:
Thank you, i'll try to do it.